app/renderer/ui/components/main/sections/attestations/index.tsx

```diff
@@ -23,6 +25,7 @@ class Attestations extends React.Component<SectionProps> {
 const AttestationsSection: SectionInfo = {
   key: "attestations",
   caption: "Attestations",
+  link: urls.ATTESTATIONS_SECTION,
```

Comment:
What do you think about rename "link" prop as "pathname"?

Reply:
I'd go for `path`.

Reply:
Path it is :+1: